src/WormholeRouter.sol

```diff
@@ -44,8 +44,15 @@ contract WormholeRouter {
     event Deny(address indexed usr);
     event File(bytes32 indexed what, bytes32 indexed domain, address data);
 
+    // --- Errors ---
+    error NotAuthorized(address sender, uint256 wards);
```

Comment:
Similar to Join. `NotAuthorized` should be just the sender.

Reply:
done in 0172295